### DISCUSSION TITLE
Documentation => Scope of Project

### DISCUSSION THREAD

**Discussion by ansibleguy** [ansibleguy76/ansibleforms#163] [Q&A] [answered]
Greetings! (:

I've looked into [your documentation](https://ansibleforms.com/introduction/) as it was mentioned [on reddit](https://www.reddit.com/r/ansible/comments/1ayw17s/).

I found that is a bit untransparent what the scope of this project/product is.

Take my questions with a bit of salt - as I only read the docs. (did not test it)

* Can it execute Ansible itself or is it 'just' a wrapper for generating commands (?)

  > ... build awesome forms, build extravars and send it to Ansible or AWX/Tower.

* What is its target audience?

  It seems the project focuses on providing a high level of customization-abilities for the forms. 

  So - in my view -  it's either for people that want more than AWX or have a very specific use-case

I'm open to collaborate with you - as I also develop a WebUI for using Ansible: https://github.com/ansibleguy/webui

Have a nice day! (:

**Comment by ansibleguy76** [comment, marked as answer]
hi there. 

it is in its core a formbuilder that builds extravars. 

it then runs local ansible playbooks and/or targets awx.

**Comment by ansibleguy76** [comment]
main goal is to extend awx surveys. 
but it comes with ansible core in case you don't have awx. 
it has joblogs, rbac, credential vault...
